pkg/util/provider/machinecontroller/machine_bootstrap_token.go

```diff
@@ -38,6 +39,8 @@ import (
 
 const placeholder = "<<BOOTSTRAP_TOKEN>>"
 
+var ignitionPlaceholder = url.QueryEscape(placeholder)
```

Comment:
```suggestion
var urlEncodedPlaceholder = url.QueryEscape(placeholder)
```

This way it would be more generic.

Reply:
Done